src/frontend/citizen-portal/src/app/api/model/ocrViolationTicket.model.ts

```diff
@@ -12,21 +12,9 @@
 import { Field } from './field.model';
 
 
-/**
- * A model representation of the extracted OCR results.
- */
 export interface OcrViolationTicket { 
-    /**
-     * Gets or sets the saved image filename.
-     */
     imageFilename?: string | null;
-    /**
-     * A global confidence of correctly extracting the document. This value will be low if the title of this   Violation Ticket form is not found (or of low confidence itself) or if the main ticket number is missing or invalid.
-     */
     globalConfidence?: number;
-    /**
-     * An enumeration of all fields in this Violation Ticket.
-     */
     fields?: { [key: string]: Field; } | null;
```

Comment:
I am curious, why did all the useful comments on this class disappear?

Reply:
so am i -- why would auto-generated code drop comments